### ISSUE TITLE
how to get scalars and metrics in real time

### ISSUE THREAD

**Issue by Liranbz** [amaiya/ktrain#312]
Hi,
Is there a way to get the metrics in real time (e.g: get the accuracy and loss value in each epoch)?
currently I use the history function, so I get the results only in the end of the training session
Thank you

**Comment by amaiya** [comment]
Hello:  You can do it however you would normally do it in Keras/TF2.  One option would be to create a [custom callback](https://keras.io/guides/writing_your_own_callbacks/) and generate whatever statistics you need with one of the `*_batch_end` methods.  All `*fit*` methods in **ktrain** accept callbacks like the tf.keras **fit**  method.
